docs/pages/docs/07-workbase/visualiser.md

```diff
@@ -0,0 +1,198 @@
+
+```graql
+match $x isa person; offset 0; limit 30; get;
+```
+
+You can change the offset and limit on the number of results as [described below](#limit-query).
+
+![Visualiser UI](/images/workbase/types_panel.png)
+
+### Save a Query
+
+If you make a query that you think you'll need to repeat regularly, and don't want to type it, or copy and paste it each time, you can save your query. The small plus sign in a circle on the right hand side of the form will bring up a tool tip, allowing you to assign it a name and save it. Saved queries can then be retrieved using the star button on the left hand side of the menu bar.
+
+![Visualiser UI](/images/workbase/save_query.png)  ![Visualiser UI](/images/workbase/starred_queries.png)
+
+### Investigate a Node
+
+A single click on any node in the knowledge graph brings up a tool tip of information about the node at the top right hand side of the screen. The information displayed includes the ID of the node, its type and any attributes associated with it, as shown in the figure below.
+
+![Single click](/images/workbase/node_panel.png)
+
+A double click on a node will bring up the relationships associated with the node, as shown below.
+
+![Double click](/images/workbase/double_click.png)
+
+Holding shift and double clicking on a node also brings up the attributes associated with it, displaying them in the knowledge graph, as shown below.
+
+![shift double click](/images/workbase/shift_click.png)
+
+You may also select multiple nodes by holding Cmd/Ctrl and clicking on nodes.
+
+### Change the Display
+
+Selecting `Node Settings` in the context menu opens the node settings panel on the lower left hand side of the screen. You can use this to show different information about a node on the graph and change the colour of the nodes.
+
+![Single click](/images/workbase/node_settings_panel.png)
+
+### Query Settings
+
+Query Settings can be accessed under the cog icon at the right hand of the horizontal icon set. The following settings are available:
+
+![Person query](/images/workbase/query_settings.png)
+
+#### Limit Query 
+
+You can change the offset and limit on the number of results returned by editing the value directly in the submission form, or by adjusting the Query Limit setting.
+
+#### Limit Neighbour  
+
+Allows the user to limit the nodes loaded when double clicking or shift + double clicking on a node.
+
+#### Autoload Role Players
+
+You can enable or disable loading role players with relationships.
+
+### Explain an Inferred Concept
+
+If a nodes is inferred, you can select the `Explain` option from the context menu and get the explainaion of how it is inferred. e.g if the cousin relationship is inferred, selecting explain would give us the following:
```

Comment:
Maybe a screenshot with context menu open on a relationship node that shows the `Explain` item in the context menu?